lib/ansible/modules/network/avi/avi_useraccount.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/python
+"""
+# Created on Aug 12, 2016
+#
+# @author: Gaurav Rastogi (grastogi@avinetworks.com) GitHub ID: grastogi23
+#
+# module_check: not supported
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: avi_useraccount
+author: Chaitanya Deshpande (chaitanya.deshpande@avinetworks.com)
+
+short_description: Avi UserAccount Module
+description:
+    - This module can be used for updating the password of a user.
+    - This module is useful for setting up admin password for Controller bootstrap.
+version_added: 2.6
+requirements: [ avisdk ]
+options:
+    old_password:
```

Comment:
This isn't clear to me.

Reply:
username and password are part of common argument spec so to update a password user has to provide new and old both passwords. Using the password field in common argument spec user will specify the new password and with the option old_password user will specify old password or default password (Avi controller image is shipped with a default password can be used once only to update the password to bootstrap)